src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java

```diff
@@ -298,7 +294,8 @@ protected void generateWork() throws IOException {
 
         protected List<PID> addChildObjects() throws IOException {
             SourceFileMapping sourceMapping = getSourceFileMapping(cdmId);
-            return Collections.singletonList(addFileObject(cdmId, cdmCreated, sourceMapping));
+            return Collections.singletonList(addFileObject(cdmId, cdmId + "/original_file",
```

Comment:
so here the `cdmId + "/original_file"` specifies the file in the work?

Reply:
I've added a comment to describe the reason for this suffix